js/components/skipControls.js

```diff
@@ -227,7 +259,7 @@ class SkipControls extends React.Component {
       buttonId === CONSTANTS.SKIP_CTRLS_KEYS.SKIP_FORWARD
     );
 
-    const isDisabled = (
+    const isDisabled = !this.props.audioOnly && (
```

Comment:
Could this be something more generic like `forceShowButtons` instead of checking for audio only?

Reply:
Yep, will add